hamocc/mo_read_ndep.F90

```diff
@@ -226,17 +227,24 @@ subroutine get_ndep(kpie,kpje,kbnd,kplyear,kplmon,omask,ndep,patmnhxdep,patmnoyd
             endif
           enddo
         enddo
-        !$OMP END PARALLEL DO
-        if (mnproc .eq. 1) then
-          write (io_stdo_bgc,*) 'iHAMOCC: getting NOy and NHx deposition from atm'
-        endif
+        !$omp end parallel do
+      else
+        !$omp parallel do private(i)
+        do j=1,kpje
+          do i=1,kpie
+            ! reduced and oxidized forms will all enter the NO3 pool
+            ndep(i,j,idepnoy) = (patmnoydep(i,j)+patmnhxdep(i,j))*fatmndep
```

Comment:
Why is here no if-block `patmnoydep(i,j) > 0. .and.  patmnhxdep(i,j) > 0.`? Either this is needed (then it should be here) or it isn't needed, but then it also could be removed in the block above for better readability and consistency?